beacon-chain/sync/topic_families_dynamic_subnets.go

```diff
@@ -0,0 +1,277 @@
+package sync
+
+import (
+	"github.com/OffchainLabs/prysm/v7/beacon-chain/p2p"
+	"github.com/OffchainLabs/prysm/v7/config/params"
+	"github.com/OffchainLabs/prysm/v7/consensus-types/primitives"
+	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/ethereum/go-ethereum/p2p/enr"
+	"github.com/pkg/errors"
+)
+
+// AttestationTopicFamily
+var _ DynamicShardedTopicFamily = (*AttestationTopicFamily)(nil)
+
+var (
+	attestationMinMeshPeers     = 8
```

Comment:
@kasey @nalepae 

I'd love feedback on:

1) What are reasonable default values for these ? Do the values I use here make sense ?
2) Do we need to make these configurable ? If yes, should we introduce a flag for these ?